brave/browser-laptop
# Refactor about:extensions with Aphrodite

## Test Plan
1. Open about:preferences#advanced
2. Disable Torrent Viewer
3. Restart the browser
4. Open about:extensions
5. Make sure Torrent Viewer is disabled (grayed out)
6. Enable Torrent Viewer
7. Make sure Torrent Viewer is enabled

## Description
Closes #7345

- extensions.less is deleted
- itemList.less is unlinked
- globalStyles.spacing.aboutPageDetailsPageWidth is introduced

Auditors: @cezaraugusto, @bsclifton


~~Blocked on #7258~~ #7258 was merged to the master
